### PR TITLE
Improve Job List Interface and Functionality

### PR DESCRIPTION
**Pull request by katsudoon1**
### Changes Overview

This PR introduces key improvements to the job list interface, focusing on enhancing its functionality and consistency. The primary addition is the "Job Created" column, which facilitates a default sorting mechanism based on job creation dates.

### Specific Changes

- **Align Job List Layout with File List (34e87fa):** Updated the job list layout to match the file list's style and structure, ensuring a consistent user experience across our platform.

- **Add "Job Created" Column (e449cc4):** Introduced a new column for displaying the creation date of each job. This addition is crucial as the job list now defaults to sorting based on the "Job Created" date, allowing users to see the most recent jobs first. This enhances the usability and management of job listings.

- **Add Sorting Functionality to Job List Table (7ef06cb):** Implemented sorting functionality, enabling users to sort jobs based on different criteria, including the newly added "Job Created" date.

- **Code Clean-Up (991090b):** Removed an unnecessary API key variable definition, making the codebase more concise and easier to maintain.

### Impact

The addition of the "Job Created" column and the default sorting by creation date significantly improve the job list's functionality. These changes make it easier for users to track and manage jobs, especially in environments with numerous job listings.

### Testing

- Visual inspection and UI comparison with the file list layout.
- Testing the new "Job Created" column for correct date display.

### CODE REVIEW

File: src/app/job-list/page.tsx [hunk 1]
```diff
@@ -74,6 +102,45 @@ const FineTuningJobsPage = () => {
     fetchJobs();
   }, []);
 
+  useEffect(() => {
+    if (sortField !== null && jobs) {
```
Comment (k-taro56) [inline src/app/job-list/page.tsx:106]: `null` チェックいります？

Comment (katsudoon1) [reply src/app/job-list/page.tsx:106]: なんか、チェックいれないとcursorで赤い波線になっちゃいました。
そのままコミットしようとしても変なエラーがでたのでChatGPTに聞いたら、これ入れとけって言われましたｗ

File: src/app/job-list/page.tsx [hunk 2]
```diff
@@ -74,6 +102,45 @@ const FineTuningJobsPage = () => {
...
+      const sortedJobs = [...jobs].sort((a, b) => {
+        if (sortField === SORT_FIELDS.JOB_ID) {
```
Comment (k-taro56) [inline src/app/job-list/page.tsx:108]: I want Compare Helper Function.

Comment (katsudoon1) [reply src/app/job-list/page.tsx]: Thanks. I fixed it.

**Review by k-taro56 — APPROVED** [review]

**Pull request merged**